contrib/pyln-proto/pyln/proto/wire.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, rawkey):
         self.key = coincurve.PrivateKey(rawkey)
 
     def serializeCompressed(self):
-        return hexlify(self.key.to_hex())
```

Comment:
this commit is unnecessary, since it reverts a change in the first commit in this same patch. You should fold these into the same commit.